distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -10,10 +10,13 @@ ALERT;Yamaha Receiver Binding: The configuration parameter names now use lower c
 
 [2.4.0]
 ALERT;Astro Binding: The 'kilometer' and 'miles' channels have been replaced by a new 'distance' channel
-ALERT;Jeelink Binding: The 'currentWatt' and 'maxWatt' channels have been replaced with 'currentPower' and 'maxPower' channels
+ALERT;Jeelink Binding: The 'currentWatt' and 'maxWatt' channels have been replaced with 'currentPower' and 'maxPower' channels.
 ALERT;WeatherUnderground Binding: A bridge has been added on top of the current things, you need to add a bridge containing your api-key.
-ALERT;ZWave Binding: Major changes have been merged to support features such as security. All things must be deleted and re-added. Refer to https://community.openhab.org/t/zwave-binding-updates/51080 for further information.
+ALERT;ZWave Binding: Major changes have been merged to support features such as security. All things must be deleted and re-added.
+ALERT;Refer to https://community.openhab.org/t/zwave-binding-updates/51080 for further information.
 ALERT;Synop Binding is now using UoM. 'wind-speed-ms' and 'wind-speed-knots' channels have been replaced by a single 'wind-speed' channel.
+ALERT;RuleEngine renamed to ruleEngine in runtime.cfg.
```

Comment:
There is no entry "RuleEngine" in runtime.cfg, so this line is pretty misleading. This parameter is in fact undocumented anyhow, so I am not sure we should mention an undocumented so prominently here.

Reply:
Well it is in use by a number of people that's in fact how I came to open this PR (right after I got hit by that Karaf auth fiasco). I believe it's always better to warn upfront rather than have people run into it (even if it does not affect many users). Changed wording to clarify it's an optional parameter.